src/main/java/com/leetcodeprep/linkedList/LinkedListCustom.java

```diff
@@ -0,0 +1,77 @@
+package com.leetcodeprep.linkedList;
+
+public class LinkedListCustom <V>{
+    private LinkedListNode head;
+    private LinkedListNode tail;
+    private int nodeCount;
+
+    public LinkedListNode add(V val){
+        if(head == null){
+            head = new LinkedListNode();
+            head.value = val;
+            tail = head;
+        }
+        else {
+            this.tail.next = new LinkedListNode();
+            this.tail.next.value = val;
+            this.tail = this.tail.next;
+        }
+        this.nodeCount++;
+        return head;
+    }
+
+    public V get(int index){
+        LinkedListNode temp = head;
+        while(index -- > 0){
+            temp = temp.next;
+        }
+        return temp.value;
+    }
+
+    public V remove(int index){
+        if(index > nodeCount || head == null){
+            throw new IndexOutOfBoundsException();
+        }
+        if(index == 0){
+            LinkedListNode t = head;
```

Comment:
Names like "t", "val" are prohibited in production code:)  And during livecoding too.
Yesterday I have found great article from ex-google engineer how he did interview.
https://www.linkedin.com/pulse/my-simple-coding-interview-question-wil-wen/ 
Why did I mention it? because naming is one of important points during coding interview:)

Reply:
When I write comment, please, try to apply it for whole code. I don't repeat comments for the same mistakes to give you chance to try to understand and to practice a little bit:)